### PR TITLE
Feature/updated profile

### PR DESCRIPTION
**Pull request by efugleberg**
# Description

Changed profile views to material UI and used Hooks to make dynamic. 
Needs some styling work yet but it's functional.

## Checklist

Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -126,7 +127,7 @@ class App extends React.Component {
             <Route path="/admin-dashboard" component={AdminDashboard} />
             <Route path="/approved-teachers" component={ApprovedTeacherList} />
             <Route path="/approved-mentors" component={ApprovedMentorList} />
-            <Route path="/view-profile" component={ViewUserProfile} />
+            <Route path="/view-profile" component={NewUserProfile} />
```
Comment (Full-lifey) [inline src/App.js]: 👍

File: src/components/views/NewUserProfile.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      maxWidth: "1140px"
+    }
+  }
+}));
+
+const NewUserProfile = props => {
+  console.log("props", props);
+  const [user, setUser] = useState({});
+
+  useEffect(() => {
```
Comment (Full-lifey) [inline src/components/views/NewUserProfile.js]: 💯

File: src/components/views/NewUserProfile.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      maxWidth: "720px"
+    },
+    "@media (min-width: 992px)": {
+      maxWidth: "960px"
+    },
+    "@media (min-width: 1200px)": {
+      maxWidth: "1140px"
+    }
+  }
+}));
```
Comment (Full-lifey) [inline src/components/views/NewUserProfile.js]: Rock on with the styles my man, that looks tough!

File: yarn.lock [hunk 4]
```diff
@@ -5479,11 +5479,6 @@ flat-cache@^2.0.1:
     rimraf "2.6.3"
     write "1.0.3"
 
-flatpickr@^4.6.2:
```
Comment (Full-lifey) [inline yarn.lock]: Finally!!!!

File: src/components/views/NewUserProfile.js (hunk 3 above)
Comment (micahjones13) [reply src/components/views/NewUserProfile.js]: ⚡️ dope

File: src/components/views/NewUserProfile.js (hunk 2 above)
Comment (micahjones13) [reply src/components/views/NewUserProfile.js]: ↪️ Nice hooks!!

File: src/components/views/NewUserProfile.js [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+    .doc(props.match.params.uid)
+    .get()
+    .then(querySnapshot => {
+      setUser(querySnapshot.data())
+    });
+  }, []);
+
+  const classes = useStyles();
+
+  if (!props.userInfo.uid && props.userInfo.awitingApproval) {
```
Comment (micahjones13) [inline src/components/views/NewUserProfile.js]: I think awaitingApproval is spelt wrong here.

**Pull request merged**